ocaml/tests/test_xapi_db_upgrade.ml

```diff
@@ -58,10 +58,9 @@ let upgrade_vm_memory_for_dmc () =
     (r.API.vM_memory_static_min <= r.API.vM_memory_static_max)
 
 let upgrade_bios () =
+  let tmp_filename = "/tmp/.previousInventory" in
```

Comment:
Using fixed names in a global directory risks conflicts. Should we not generate a file name?

Reply:
The issue is that both the writer (test) and the reader (db_upgrade) have to use the same filename. I've tried to inject the value of the generated filename to the db upgrade function using the current interface to no avail. I wanted to avoid being too disruptive around that code.

Reply:
Maybe not use a hidden file? We would never look for it.